bcipy/helpers/system_utils.py

```diff
@@ -43,6 +43,12 @@ def is_connected(hostname: str = "1.1.1.1", port=80) -> bool:
     return False
 
 
+def is_battery_powered() -> bool:
+    """Check if this current computer is a laptop currently using its battery."""
```

Comment:
Some more context may be useful! `Returns True if the computer is currently running on battery power. This can impact the performance of hardware (ex. GPU) needed for BciPy operation by entering power saving operations.`